R/aaa.R

```diff
@@ -0,0 +1,5 @@
+# Register cpp -----------------------------------------------------------------
+
+#' @useDynLib xtensor, .registration = TRUE
+#' @importFrom Rcpp sourceCpp
+NULL
```

Comment:
Is there really an interest in using Roxygen in this case?

At the moment, the documentation is hosted on readthedocs and generated from the doxygen anotations of `xtensor-r`.

Maybe there is a means to extract the same information for the R package documentation?

Reply:
> white_check_mark I removed the use of roxygen2. If you ever decide to have some R functions hosted in this xtensor package that demonstrate any features, you might want to reconsider using roxygen2. It makes life so much simpler when documenting them over doing it the manual way. But for now it's not a problem.
>

Yeah, I think it is especially interesting if there is actual R code to document.

> I'm not sure this package needs the readthedocs documentation (not even sure there would be a good automated way to get it in a format that R would like). It might be useful to point to the readthedocs url somewhere though, if you aren't doing so already.

There is already documentation on readthedocs for

- [xtensor](xtensor.readthedocs.io)
- [xtensor-blas](xtensor-blas.readthedocs.io)
- [xtensor-python](xtensor-python.readthedocs.io)
- [xtensor-julia](xtensor-julia.readthedocs.io)
- [xtensor-io](xtensor-io.readthedocs.io)
- [xtensor-r](xtensor-r.readthedocs.io)

so I think that it makes sense to make the documentation available there, even if it is also included in the R package.